containers/ws/README.md

```diff
@@ -1,99 +1,153 @@
-# Cockpit on Fedora CoreOS or other container hosts
+# Cockpit WS Container
 
-The standard Fedora and Red Hat Enterprise Linux CoreOS images does not contain
-Cockpit packages.
+This container is suitable for scenarios where you can not or wish not to deploy the Cockpit Web Service and browser
+application natively.
 
-1. Install Cockpit packages as overlay RPMs:
+## Tags
```

Comment:
Adding this section should happen in the previous commit which adds these builds/tags, as these belong together. The other README changes should indeed stay a separate commit.